DevSkim-DotNet/Microsoft.DevSkim.CLI/Commands/SuppressionCommand.cs

```diff
@@ -0,0 +1,126 @@
+                    var potentialPath = Path.Combine(_opts.Path, fileName.OriginalString);
+                    // Flatten all the replacements into a single list
+                    var listOfReplacements = resultGroup
+                      .Where(x => x.Locations is { })
+                      .SelectMany(x => x.Locations).Select(x => x.PhysicalLocation).ToList();
+
+                    listOfReplacements
+                    .Sort((a, b) => a.Region.StartLine - b.Region.StartLine);
+
+                    var distinctReplacements  = listOfReplacements
+                    .GroupBy(x => x.Region.StartLine)
+                    .Select(x => x.FirstOrDefault());
+
+                    var ruleIds = string.Join(",", resultGroup.Select(x => x.RuleId).Distinct());
+
+                    if (File.Exists(potentialPath))
+                    {
+                        var theContent = File.ReadAllText(potentialPath).Split(Environment.NewLine);
+                        int currLine = 0;
+                        var sb = new StringBuilder();
+                        foreach (var replacement in distinctReplacements)
+                        {
+                            var zbStartLine = theContent[0] == string.Empty ? replacement.Region.StartLine: replacement.Region.StartLine - 1;
+                            var isMultiline = theContent[zbStartLine].EndsWith(@"\");
+                            var ignoreComment = $"{getPrefixComment(replacement.Region.SourceLanguage)} DevSkim: ignore {ruleIds} {getSuffixComment(replacement.Region.SourceLanguage)}";
+                            
+                            foreach (var line in theContent[currLine..zbStartLine])
+                                    sb.Append($"{line}{Environment.NewLine}");
+                            
+                            var suppression = isMultiline ? $"{ignoreComment}{theContent[zbStartLine]}{Environment.NewLine}" : 
+                            $"{theContent[zbStartLine]}{ignoreComment}{Environment.NewLine}";
+                            sb.Append(suppression);
+
+                            currLine = zbStartLine + 1;
+                        }
+
+                        foreach (var line in theContent[currLine..^1])
+                            sb.Append($"{line}{Environment.NewLine}");
+                        sb.Append($"{theContent.Last()}");
+
+                        if (!_opts.DryRun)
+                        {
+                            File.WriteAllText(potentialPath, sb.ToString());
+                        }
+                    }
+                    else
+                    {
+                        Console.Error.WriteLine($"{potentialPath} specified in sarif does not appear to exist on disk.");
+                    }
+                }
+            }
+
+            return (int)ExitCode.NoIssues;
+        }
+
+        private string getInlineComment(string language)
+        {
+           Languages devSkimLanguages = DevSkimLanguages.LoadEmbedded();
```

Comment:
Please refactor this to load the languages once in the ctor and then call the devSkimLanguages apis directly instead of using these helper methods.

Consider supporting custom languages/comments similar to the extant Analyze/Fix commands. I can make that change later if that's not something you want to handle.

Reply:
I'm not sure I'm understanding how to support custom languages in this case

Reply:
My mistake. This appears to be a functionality gap between Application Inspector and DevSkim. Don't worry about it at this time in that case, I'll add this functionality throughout later.